games-arcade/jumpnbump/jumpnbump-1.61.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit python-single-r1
+
+DESCRIPTION="A funny multiplayer game about cute little fluffy bunnies"
+HOMEPAGE="https://libregames.gitlab.io/jumpnbump"
+SRC_URI="https://gitlab.com/LibreGames/${PN}/uploads/95acdae2a232513f068e260977371dcf/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+REQUIRED_USE="gtk? ( ${PYTHON_REQUIRED_USE} )"
+
+DEPEND="
+	media-libs/sdl2-mixer[mod]
+	media-libs/libsdl2
+	media-libs/sdl2-net
+"
+RDEPEND="${DEPEND}
+	gtk? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep '
+			dev-python/pygobject[${PYTHON_USEDEP}]
+		')
+		x11-libs/gdk-pixbuf:2[introspection]
+		x11-libs/gtk+:3[introspection]
+	)
+"
+
+src_compile() {
+	emake PREFIX=/usr
```

Comment:
Could you use ${EPREFIX}/usr?

Reply:
Fixed.

I also changed `${D}` to `${ED}`, let me know if that's wrong: https://github.com/gentoo/gentoo/pull/17886/commits/47f1a0ab74d03ade3582e51e9cfc207390447e75#diff-ba36f3cd0808c650dfa5029505e26ba1097f732a9613a3cc5e4d34626b498fa9R43